.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: ruby
+rvm:
+- 2.4.1
+env:
+- DB=postgresql
+script:
+- RAILS_ENV=test bundle exec rake --trace db:create
+- RAILS_ENV=test bundle exec rake --trace db:migrate
+- bundle exec rake db:test:prepare
+- bundle exec rspec spec/
+- bundle exec rubocop --config .rubocop.yml
+deploy:
+  provider: heroku
+  api_key:
+    secure: tHFmlbx/IbkH/5K+Cf7ecZbGXRpZ+ninqf/L7H5cUm0SYWtdesC1KZMzU/Z70U8uySYrjf1pAowcgxi+JtY6Pqy4fPguZff2UTRJXNQltHt5eui0oYz7kNg2xzHj/guoi/rkuwF9lhZ4n5I6Vakg4PU2Ss29TqcNOnrw8vnYl8OJbQhmb28ymD4AItiXI354+VAX16ot9H2aDTr5qfjAkoed/3lKJFlb9xTKZWsSqYR+BPzcQSDeQZNupDIV9Yrg4FptneAQ84xRoI06n8EEVefSEKenL+MLKawy3LbdhiGGr8sO5KAlJoRUoKabyFbT9/CNu1FxtxHbuu8UXdeUC3jrqbI44+V20o/AF+Lufm5+4dzv1bw3Wjn46qvElCdSZnCrm/RfBqTYtlI6vxcnHgsOl7LTlyQJgOoygqZyNfbADKEk08Rj5wGYFMnwgSSgx/ZWY82VMb9PsJnzBCPsyYX5oU7jj7KrINiKvrM8adzOTeTGb17IzFCRiimI4jzDUoLSDRvB1yGzYgJ/GxAIif/wrMRDelWKUq1X4vsM23J/m7AWMmftWg7lvSs3+q0kq1GDOUnMKQnq9x5jqyyRsMIob4wza1xUouHTYMLDgK86iAaew6Nm4p6Pq8JyU+6extgHcuQpryzmkyh+bli4DJnRHgiVFL0qX9amrgP4gbA=

```

Comment:
:+1: If you had your non-encrypted API key in here before, you should regenerate a new one (since this is a public repo). Safer to make sure that old API key doesn't work anymore.